styles/planet-minecraft/catppuccin.user.css

```diff
@@ -128,6 +128,15 @@
       }
     }
 
+    .content-actions li, a:hover {
+      background: @mantle !important;
+      border-color: @mantle !important;
+    }
+
+    .content-actions li a {
+      color: @text !important;
+    }
+    
```

Comment:
There are tons of tohher `.content-actions` sections in this userstyle, some of which appear to have been for theming this buttons in the past. Can you consolidate and nest the `.content-actions` sections into one?

Reply:
sure